draft-ietf-quic-transport.md

```diff
@@ -2353,7 +2351,8 @@ processed successfully.  The timer is also restarted when sending an
 ack-eliciting packet (see {{QUIC-RECOVERY}}), but only if no other ack-eliciting
 packets have been sent since last receiving a packet.  Restarting when sending
 packets ensures that connections do not prematurely time out when initiating new
-activity.
+activity.  An endpoint should be careful to avoid an idle timeout if it is idle
+because of flow control (see {{data-flow-control}}).
```

Comment:
Maybe change the reference to {{flow-control}}? Otherwise, blocked by stream concurrency would be out-of-scope.

Or, we might say "data flow control ({{data-flow-control}}) or stream concurrency ({{controlling-concurrency}})."